### PR TITLE
Update isolation guidance with new CDC guidelines

### LINKED ISSUE #3199: Update Isolation Guidance Included When Test Results Are Available
Under the section "What does my result means?", the isolation guidance needs to be updated to reflect the shorter isolation guidance CDC announced at the end of December. Thanks!

cc- @LGFranklin @tim-skylight

### PR DESCRIPTION
**Pull request by emmastephenson**
## Related Issue or Background Info

- Fixes #3199 

## Changes Proposed

- This is a temporary fix; we need both better Spanish translations and likely a look at the language in general to make sure we link to the updated quarantine guidance.

## Screenshots
Screen Shot 2022-01-13 at 1 58 58 PM

### Infrastructure
- [x] **Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!**

### Design
- [x] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [x] Any content changes (including new error messages) have been approved by content team

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Review by nickclyde — DISMISSED** [review]
:+1:

File: frontend/src/lang/en.ts [hunk 1]
```diff
@@ -267,7 +267,9 @@ export const en = {
           guidelines: {
             li0: "Stay home when you are sick, except to get medical care.",
             li1:
-              "Self isolate for 10 full days after symptoms first appeared (or starting the day after you had your test, if you have no symptoms).",
+              "Stay home for 5 days." +
```
Comment (zdeveloper) [inline frontend/src/lang/en.ts:270]: missing a space after the periods
image

File: frontend/src/lang/es.ts [hunk 2]
```diff
@@ -281,7 +281,9 @@ export const es: LanguageConfig = {
             li0:
               "Quédese en casa cuando esté enfermo, excepto para recibir atención médica.",
             li1:
-              "Autoaíslese durante 10 días completos después de que aparezcan los síntomas (o a partir del día siguiente a la realización de la prueba, si no tiene síntomas).",
+              "Quédate en casa durante 5 días." +
```
Comment (zdeveloper) [inline frontend/src/lang/es.ts:284]: missing space after periods, maybe, idk if thats the case for spanish

image

File: frontend/src/lang/en.ts (hunk 1 above)
Comment (zdeveloper) [reply frontend/src/lang/en.ts:270]: PDF:
image

**Review by zdeveloper — CHANGES_REQUESTED** [review]

**Comment by emmastephenson** [comment]
> 

@Rebecca-LM The revised language looks good, thanks! I can add something about checking the CDC website and following local guidelines, do you have any preference on where that goes? We could add it to the end of the guideline list if that works for you (after "Have a supply of clean, disposable face masks"). Probably best to add a new paragraph altogether, but that will have to wait for later as it requires us to update the JS that generates the PDF and patient result.

File: frontend/src/lang/en.ts (hunk 1 above)
Comment (emmastephenson) [reply frontend/src/lang/en.ts]: Good catch, thank you! Updated:
Screen Shot 2022-01-13 at 9 49 28 AM

**Comment by Rebecca-LM** [comment]
> > 
> 
> @Rebecca-LM The revised language looks good, thanks! I can add something about checking the CDC website and following local guidelines, do you have any preference on where that goes? We could add it to the end of the guideline list if that works for you (after "Have a supply of clean, disposable face masks"). Probably best to add a new paragraph altogether, but that will have to wait for later as it requires us to update the JS that generates the PDF and patient result.

@emmastephenson if the below screenshot still reflects the format we're using for pos results, then yes, below the bullet that references face masks in the home I'd like to add/modify to: "More information is available at cdc.gov/coronavirus/2019-ncov/if-you-are-sick. Local laws may vary, always check your state and county health department guidelines.

Screen Shot 2022-01-13 at 9 36 30 AM

**Comment by emmastephenson** [comment]
@Rebecca-LM sounds good, updated with the new language!

The most canonical source of truth is this file: https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/lang/en.ts
But as you can see it's not formatted the same at all; it just contains the variables we use in the PDF and patient results. The easiest way to see an up-to-date version is probably by printing a result on the `demo` or `training` site. We can look into making a story for this in Chromatic if that's helpful too.

**Review by zdeveloper — APPROVED** [review]
LGTM!

**Review by nickclyde — APPROVED** [review]

**Comment by Rebecca-LM** [comment]
@nickclyde are you all using google translate for these? are you able to update the spanish version with the changes if so?

**Comment by emmastephenson** [comment]
@Rebecca-LM yes, I used google translate for these - we needed to get something out ASAP, so unfortunately didn't have time to wait for the CDC translation team :/ I'm happy to make any changes though!

**Comment by Rebecca-LM** [comment]
@emmastephenson that's totally fine with me (google translate is pretty good these days). i just wanted to double check the changes we made today will be reflected in the spanish version too!

**Comment by emmastephenson** [comment]
@Rebecca-LM oh I see, yes I also made changes to the Spanish version! Just didn't take a screenshot.

**Pull request merged**